src/CodeGenerator.php

```diff
@@ -127,8 +127,10 @@ private static function generatePackage(
         ?string $gapicYaml,
         ?string $serviceYaml
     ) {
-        $version = Helpers::nsVersion($namespace);
-        $version = is_null($version) ? '' : $version . '/';
+        $version = Helpers::nsVersionAndSuffixPath($namespace);
+        if ($version !== '') {
```

Comment:
Maybe add a brief comment on the intended behavior? After our discussion, I think I see what's going on, but a future reader may not.

Reply:
done